docs/eps/DONE_DEFINITION.md

```diff
@@ -0,0 +1,22 @@
+## Histórico de Revisões
+
+| Data | Versão | Descrição | Autores |
+| --- | --- | --- | --- |
+| 27/03/2018 | 0.0.1 | Criação do documento | João Pedro Sconetto |
+
+---
+
+# Introdução
+Este documento visa esclarecer entre os envolvidos no projeto _Dr. Down_ dúvidas que possam surgir sobre quando uma tarefa será considerada pronta, quais são os critérios levados em consideração e quais destes devem ser cumpridos para ser aceita como __pronto__ no viés do projeto.
+
+# História
+Uma __história__ será considerada pronta quando a codificação e/ou alteração necessária estiver completa, testada com cobertura de código (implementando o testes unitários, funcionais e de integração) visando manter a mesma igual ou acima de 90% e a documentação do projeto (caso se necessário modificação) atualizada.
+
+# _Feature_
+Uma **_feature_** será considerada pronta quando todas as histórias derivadas da mesma estarem entregues, prontas e testadas de acordo com os critérios supracitados.
+
+# Sprint
+Uma **_sprint_** será considerada 'pronta' quando as histórias que foram planejadas para execução dentro do _time box_ da mesma estiverem entregues, prontas e testadas de acordo com os critérios supracitados, sendo importante frisar de que a entrega deverá ser feita via **_Pull Request_** no repositório oficial do projeto e o mesmo deverá ser discutido, testado e aprovado por ao menos 2 membros de EPS (preferência para a aprovação do _Scrum Master_ e do _Product Owner_).
```

Comment:
Como assim, Sprint pronta? Pesquisei sobre esse conceito e não achei nada a respeito. Pelo que li, a Sprint não fica pronta. Ela apenas é automaticamente finalizada quando acaba o _time box_ definido pra ela ou, quando por algum motivo de força maior, ela é cancelada.

Além disso, como mencionei no comentário sobre no tópico sobre histórias, o que é mencionado sobre PR e a aprovação necessária do _Product Owner_ (sendo que este aprova ou não de acordo com os critérios de aceitação definidos para a história) deve estar no tópico história.

Reply:
A ideia de definir uma sprint como 'pronta' (finalizada) é para reforçar para os membros de desenvolvimento que a sprint pode ser desenvolvida antes do tempo hábil do _time box_, e novamente, o intuito deste documento é reforçar o que já foi elaborado no documento da metodologia.

Reply:
Entendi a sua intenção agora, @sconetto . Então sugiro uma alteração no começo desse texto, para deixar mais clara a ideia:
"Uma **_sprint_** poderá ser finalizada/pronta antes do término de sua _time box_, desde que todas as histórias que foram planejadas para serem entregues durante a _sprint_ estiverem entregues, (...)"